recipes/C-iTRACE/recipe.py

```diff
@@ -0,0 +1,24 @@
+from pangeo_forge_recipes.patterns import ConcatDim, FilePattern, MergeDim
+from pangeo_forge_recipes.recipes.xarray_zarr import XarrayZarrRecipe
+
+# C-iTRACE covers a number of proxies
+variables = ['d18O']
+
+def make_url(time, variable):
+    url = 'https://gdex.ucar.edu/dataset/204_ajahn/file/ctrace.decadal.{variable}.nc'.format(variable=variable)
+    return url
+
+# there is only one, but notably ~6 Gb file covering the full timeseries
```

Comment:
```suggestion
# there is only one, but notably ~6 GB file covering the full timeseries
```

Reply:
☝️ This is just a very small change to re-trigger the Pangeo Forge Cloud CI system (which runs on every commit)